docs/modules/ROOT/pages/node_pattern.adoc

```diff
@@ -3,7 +3,7 @@
 Node pattern is a DSL to help find specific nodes in the Abstract Syntax Tree
 using a simple string.
 
-It reminds the simplicity of regular expressions but used to find specific
+It resembles the simplicity of regular expressions but used to find specific
```

Comment:
I guess it was supposed to be this?

```suggestion
It reminds of the simplicity of regular expressions but used to find specific
```

Reply:
`It reminds of the` is not clear English. 
Adding the 2nd `is` does make sense though.

Reply:
updated the PR w/ the change

Reply:
You might say:

`It reminds one of the simplicity..` or `It may remind you of the simplicity`... but that sounds too informal.